ember/app/ui/work-items/controller.js

```diff
@@ -0,0 +1,81 @@
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { tracked } from "@glimmer/tracking";
+import { queryManager } from "ember-apollo-client";
+import calumaQuery from "ember-caluma/caluma-query";
+import { allWorkItems } from "ember-caluma/caluma-query/queries";
+import { restartableTask } from "ember-concurrency-decorators";
+
+export default class WorkItemsIndexController extends Controller {
+  queryParams = ["order", "responsible", "status", "role"];
+
+  @service session;
+
+  @queryManager apollo;
+
+  @calumaQuery({ query: allWorkItems, options: "options" })
+  workItemsQuery;
+
+  // Filters
+  @tracked order = "urgent";
+  @tracked responsible = "all";
+  @tracked status = "open";
+  @tracked role = "active";
+
+  get options() {
+    return {
+      pageSize: 20,
+    };
+  }
+
+  get columns() {
+    return [
+      "task",
+      "instance",
+      "description",
+      ...(this.status === "open"
+        ? ["deadline", "responsible"]
+        : ["closedAt", "closedBy"]),
+    ];
+  }
+
+  @restartableTask
+  *fetchWorkItems() {
+    const filter = [];
```

Comment:
How about initializing the filter with the "default" options so all the `else`-cases are not needed?

Reply:
As discussed: The `else`-part is ok but check if an empty filter is even needed.

Reply:
Removed useless else